### PR TITLE
[#4383] REPF no longer executes rule code as a microservice on continuation (master)

### PR DESCRIPTION
**Pull request by korydraughn**
- [Passed CI](http://172.25.14.63:8080/view/Personal/job/irods-build-and-test-workflow/2052/)
- Passed Docker

### CODE REVIEW

**Comment by trel** [comment]
i don't see anything missing/incorrect.

adds the catch for `last_error_code` and explains the why to the next soul who goes looking.

i like testing via the 'trace' level - i think that's a first.

**Comment by korydraughn** [comment]
This code has not been tested in docker or CI.

File: scripts/irods/test/test_rule_engine_plugin_framework.py [hunk 1]
```diff
@@ -23,6 +24,181 @@
...
+
+    def setUp(self):
+        super(Test_Rule_Engine_Plugin_Framework, self).setUp()
+        self.admin = self.admin_sessions[0]
+
+    def tearDown(self):
+        super(Test_Rule_Engine_Plugin_Framework, self).tearDown()
+
+    @unittest.skipIf(plugin_name == 'irods_rule_engine_plugin-python' or test.settings.RUN_IN_TOPOLOGY, "Skip for Python REP and Topology Testing")
+    def test_continuation_causes_NO_MICROSERVICE_FOUND_ERR__issue_4383(self):
```
Comment (trel) [inline scripts/irods/test/test_rule_engine_plugin_framework.py]: test_continuation_does_not_find_NO_MICROSERVICE_FOUND_ERR__issue_4383

Comment (korydraughn) [reply scripts/irods/test/test_rule_engine_plugin_framework.py]: will change

**Pull request merged**